docs/contributing-testing.rst

```diff
@@ -27,14 +27,38 @@ of code organization.
 How do I run the test suite?
 ----------------------------
 
-With `docker <https://www.docker.com/products/docker>`_ installed, run this.
+We assume you have `docker <https://www.docker.com/products/docker>`_ installed.
+
+In addition, you will need `riot <https://ddriot.readthedocs.io/en/latest/>`_ and `hatch <https://hatch.pypa.io/latest/>`_.
+
+.. code-block:: bash
+
+    $ pip install riot==0.17.4
+    $ pip install hatch==1.7.0
+
+Our test environments are managed with riot. However, we are migrating to hatch.
+
+For riot environments, you can run:
 
 .. code-block:: bash
 
     $ scripts/ddtest riot run -p 3.10
 
 This command runs the entire test suite, which is probably not what you want to do.
 
+For hatch environments, you can run:
```

Comment:
This isn't in scope for this pull request, but the two paths in this documentation suggest that it could be helpful to provide a script that makes the hatch/riot decision so contributors don't have to.